release-please-config.json

```diff
@@ -6,6 +6,9 @@
     },
     "packages/release-brancher": {
       "release-type": "node"
+    },
+    "packages/mono-repo-publish": {
```

Comment:
we're going to use mono-repo-publish to publish the newest version of mono-repo-publish 🤯